staticaddr/withdraw/manager.go

```diff
@@ -315,6 +381,41 @@ func (m *Manager) WithdrawDeposits(ctx context.Context,
 		return "", "", err
 	}
 
+	published, err := m.publishFinalizedWithdrawalTx(ctx, finalizedTx)
+	if err != nil {
+		return "", "", err
+	}
+
+	if !published {
+		return "", "", nil
+	}
+
+	// If this is the first time this cluster of deposits is withdrawn, we
+	// start a goroutine that listens for the spent of the first input of
+	// the withdrawal transaction.
+	// Since we ensure above that the same ensemble of deposits is
+	// republished in case of a fee bump, it suffices if only one spent
+	// notifier is run, since if the first input is spent.
```

Comment:
What happens if `loopd` restarts after first withdrawal tx is published? Will it listen for spending after the restart?

Reply:
yes, after a restart `recoverWithdrawals` will republish the latest version of the withdrawal tx and monitor the spending.